Brain Blocks

```diff
@@ -1,4 +0,0 @@
-Front
```

Comment:
Body Movement
Intelligence, concentration, self awareness
Judgement, planning, problem solving
Personality, behavior, emotions
Speech, speaking and writing